subtest/scenario.py

```diff
@@ -14,19 +14,25 @@
 from loguru import logger
 import operator
 import random
+from pathlib import Path
 
-from dataset import Dataset
-import constants
-from partner import Partner
+from .dataset import Dataset
+from . import constants
+from .partner import Partner
+from . import multi_partner_learning
+from . import contributivity
 
 
 class Scenario:
-    def __init__(self, params, experiment_path, scenario_id=1, n_repeat=1, is_dry_run=False):
+    def __init__(self, params, experiment_path=Path(r"./experiments/nlp"), scenario_id=1, n_repeat=1, is_dry_run=False):
 
         # ---------------------------------------------------------------------
         # Initialization of the dataset defined in the config of the experiment
         # ---------------------------------------------------------------------
 
+        # List of all partners defined in the scenario
+        self.partners_list = []
+
```

Comment:
Why move it in the "Initialization of the dataset defined in the config of the experiment" section instead of the "Definition of collaborative scenarios" section which was clearer?

Reply:
For some reasons, it wouldn't work otherwise.

I had to manually initiate it in every notebook before every `run_scenario()` call.

By moving this above it fixes the issue but the code is less clear.

Maybe we should totally swap the 2 portion of the code: 
```
        # --------------------------------------
        #  Definition of collaborative scenarios
        # --------------------------------------
```

and:


```
        # ---------------------------------------------------------------------
        # Initialization of the dataset defined in the config of the experiment
        # ---------------------------------------------------------------------
```